src/Microsoft.ML.StandardLearners/Standard/SdcaBinary.cs

```diff
@@ -1649,8 +1649,22 @@ public sealed class Options : BinaryOptionsBase
             /// <value>
             /// If unspecified, <see cref="LogLoss"/> will be used.
             /// </value>
-            [Argument(ArgumentType.Multiple, HelpText = "Loss Function", ShortName = "loss", SortOrder = 50)]
-            public ISupportSdcaClassificationLossFactory LossFunction = new LogLossFactory();
+            [Argument(ArgumentType.Multiple, Name = "LossFunction", HelpText = "Loss Function", ShortName = "loss", SortOrder = 50)]
+            internal ISupportSdcaClassificationLossFactory LossFunctionFactory = new LogLossFactory();
+
+            /// <summary>
+            /// The custom <a href="tmpurl_loss">loss</a>.
```

Comment:
>tmpurl_loss [](start = 36, length = 11)

What dose it link to?

Reply:
Shahab is working on this, issue #2356

---
In reply to: [262720628](https://github.com/dotnet/machinelearning/pull/2856#discussion_r262720628) [](ancestors = 262720628)